charts/agent/templates/configmap.yaml

```diff
@@ -26,6 +26,18 @@ data:
 {{- if .Values.global.sysdig.tags }}
     tags: {{ include "agent.tags" . }}
 {{- end }}
+{{- if or (.Values.proxy.httpProxy | default .Values.global.proxy.httpProxy) (.Values.proxy.httpsProxy | default .Values.global.proxy.httpsProxy) }}
```

Comment:
This check is not the same as the one in the deployment ConfigMap. Is there a reason for that difference?